src/Page/Shared/Data.elm

```diff
@@ -58,7 +59,7 @@ videoDecoder =
                 |> Json.Decode.andThen idToEmbedSrc
             )
         )
-        (Json.Decode.field "description" Json.Decode.string)
+        (Json.Decode.field "description" Json.Decode.string |> Json.Decode.Extra.withDefault "")
```

Comment:
Is this fields optional or required in the CMS? We've been using `Maybe.withDefault` for optional fields.

Reply:
I don't think that when new fields are added to the CMS all the files in the CMS get that field populated so some of the guides may be missing the field. But then someone may edit a file and remove the description- how does the CMS represent missing values? does it blank the value or remove the key entirely?

so now we parse this field that may or may not be there and may or may not have a value. we could use a `maybe` on that but we'll still have to check that a value actually exists in the front end. so instead we just give it a default value so it is always a string and just check against that, which i think is a little simpler

Reply:
What I meant is that we are codifying optional fields as Maybe and required fields as assumed to be there - so that the build would break if they are not there. If a new required field is added we need to add it to all the content with the PR.

Reply:
Otherwise you are not getting the garuntee that the data is in the shape we expect it to be in.

Reply:
But I thought video descriptions (the value) are optional? we'd have to parse the (maybe present) value, check if it is empty and return `Nothing` if it is blank?